### PR TITLE
Track donate button taps on Participate page

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1843 

URL for the donate button will be updated (to `https://donate.mozilla.org?utm_source=foundation.mozilla.org&utm_medium=referral&utm_content=participate`) once this has been pushed to production.

### CODE REVIEW

File: source/js/analytics.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import ReactGA from 'react-ga';
```
Comment (mmmavis) [inline source/js/analytics.js]: `ReactGA` not being used anywhere in this file

**Comment by mmmavis** [comment]
@alanmoo let me know if looking at the code is enough for you to review this PR. If not, I'll manually create a Participate page that is identical to what we have on production.

File: source/js/main.js [hunk 2]
```diff
@@ -203,6 +203,16 @@ let main = {
         Analytics.sendGAEvent(`navigation`, `page footer cta`, label);
       });
     }
+
+    if (document.querySelector(`#view-participate .card-cta .btn[href*="donate.mozilla.org"]`)) {
+      document.querySelector(`#view-participate .card-cta .btn[href*="donate.mozilla.org"]`).addEventListener(`click`, () => {
```
Comment (alanmoo) [inline source/js/main.js]: Can this be saved as a variable so we don't have to fetch it twice? Not necessarily from a performance standpoint, but from a readability one.

Comment (mmmavis) [reply source/js/main.js]: Yea I thought about doing this but decided to go with what we already have in the rest of the code. (No variable is declared for any DOM nodes.) Maybe we should file a followup ticket to tidy things up all at once?

**Comment by Pomax** [comment]
looks like this still needs that refactor to cache the query selector result, @mmmavis?

**Comment by Pomax** [comment]
looks like this still needs that refactor to cache the query selector result, @mmmavis?

**Pull request merged**